packages/core/database/lib/dialects/mysql/database-inspector.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const { MARIADB, MYSQL } = require('../../utils/constants');
+
+const SQL_QUERIES = {
+  VERSION: `SELECT version()`,
```

Comment:
can you alias it to avoid the `['version()']` line 16?

Reply:
done